src/vochain/vochain.proto

```diff
@@ -5,71 +5,74 @@ package dvote.types.v1;
 option go_package = "go.vocdoni.io/proto/build/go/models";
 
 message VoteEnvelope {
-        bytes nonce = 1;  // Unique number per vote attempt, so that replay attacks can't reuse this payload
-        bytes processId = 2;  // The process for which the vote is casted
-        Proof proof = 3;  // Franchise proof
-        bytes votePackage = 4;   // JSON string of the Vote Package (potentially encrypted), encoded as bytes.
-
-        // optional bytes nullifier = 5;
-        bytes nullifier = 5;  // Hash of the private key + processId
-
-        repeated uint32 encryptionKeyIndexes = 6; // On encrypted votes, contains the (sorted) indexes of the keys used to encrypt
+	// Unique number per vote attempt, so that replay attacks can't reuse this payload
+	bytes nonce = 1;
+	// The process for which the vote is casted
+	bytes processId = 2;
+	// Franchise proof
+	Proof proof = 3;
+	// JSON string of the Vote Package (potentially encrypted), encoded as bytes.
+	bytes votePackage = 4;
+	// Hash of the private key + processId
+	bytes nullifier = 5;
+	// On encrypted votes, contains the (sorted) indexes of the keys used to encrypt
+	repeated uint32 encryptionKeyIndexes = 6;
 }
 
 message Census {
-        enum Type {
-                UNKNOWN = 0;
-                ARBO_BLAKE2B = 1;
-                ARBO_POSEIDON = 2;
-                ETHEREUMSTORAGE = 3;
-                ETHEREUMACCOUNT = 4;
-                CA = 5;
-                GRAVITON = 1001;
-                IDEN3 = 1002;
-        }
+	enum Type {
+		UNKNOWN = 0;
+		ARBO_BLAKE2B = 1;
+		ARBO_POSEIDON = 2;
+		ETHEREUMSTORAGE = 3;
+		ETHEREUMACCOUNT = 4;
+		CA = 5;
+		GRAVITON = 1001;
+		IDEN3 = 1002;
+	}
 }
 
 message Proof {
-        oneof payload {
```

Comment:
Why all these changes??

Reply:
because I wanted to unify the padding, in some places we use 4 spaces in others 8 spaces and in others 3, i think this way is cleaner and does not affect the commit while improves styling

Reply:
I'd say: do it on a separate commit please... else its hard to identify what the commit changes. However I understand now going back its a mess, so let's move forward.